src/ppom-controller.test.ts

```diff
@@ -593,11 +618,14 @@ describe('PPOMController', () => {
       jest.useFakeTimers().setSystemTime(new Date('2023-01-04'));
       callBack({ providerConfig: { chainId: '0x6' } });
 
-      expect(Object.keys(ppomController.state.chainStatus)).toHaveLength(5);
+      jest.useFakeTimers().setSystemTime(new Date('2023-01-04'));
```

Comment:
I needed to add one more because the condition in ppomController for this check to happen is > 5. It doesn't do the check if its <= 5